migrate.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+source venv/bin/activate
+
+until echo > /dev/tcp/$POSTGRES_HOST/5432; do sleep 1; done
```

Comment:
this will break when we switch to RDS unless the additional env variable is added there (it's a variable needed by the `postgres` container more than by this one), but we'll deal with it when necessary